web/source/osk/oskRow.ts

```diff
@@ -69,27 +69,31 @@ namespace com.keyman.osk {
 
     public refreshLayout(vkbd: VisualKeyboard, rowHeight: number, bottom: number, pad: number) {
       const rs = this.element.style;
-      if(!vkbd.isStatic) {
-        rs.bottom=bottom+'px';
+      if(vkbd.usesFixedHeightScaling) {
```

Comment:
As mentioned for oskLayer.ts, this function now handles both desktop and touch layout code, not just touch-only code.  It's one step closer to merging the two layout code paths.